commit 4e5b98d3d928579bc3530893ceaf166337ea8e22
Author: Tom Eastep <teastep@shorewall.net>
Date:   Fri Dec 29 15:32:05 2017 -0800

    Only search files in each CONFIG_PATH directory - no recursion
    
    Signed-off-by: Tom Eastep <teastep@shorewall.net>

diff --git a/Shorewall/lib.cli-std b/Shorewall/lib.cli-std
index 6f220aa3d..95eafdf65 100644
--- a/shorewall-5.1.10.1/lib.cli-std
+++ b/shorewall-5.1.10.1/lib.cli-std
@@ -386,7 +386,7 @@ uptodate() {
     IFS=':'
 
     for dir in $g_shorewalldir $CONFIG_PATH; do
-	if [ -n "$(find ${dir} -newer $1)" ]; then
+	if [ -n "$(find ${dir} -mindepth 1 -maxdepth 1 -newer $1)" ]; then
 	    IFS="$ifs"
 	    return 1;
 	fi
